wp-content/themes/mambaone/single-cpt_lessons.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*Template Name: Single cpt Lessons
+*/
+
+get_header();
+
+$args = array(
```

Comment:
It is always important to query only the published posts, unless otherwise required.